receiver/awscontainerinsightreceiver/README.md

```diff
@@ -819,12 +825,6 @@ kubectl apply -f config.yaml
 | container_status_running                                      | Count        |
```

Comment:
similarly container_status_running, container_status_terminated, container_status_waiting should be changed to pod_container_status_running, pod_container_status_terminated, pod_container_status_waiting